taplo.toml

```diff
@@ -0,0 +1,37 @@
+## https://taplo.tamasfe.dev/configuration/#configuration-file
+
+include = ["**/Cargo.toml"]
+exclude = ["target/**/Cargo.toml"]
+
+[formatting]
+# Align consecutive entries vertically.
+align_entries = false
+# Append trailing commas for multi-line arrays.
+array_trailing_comma = true
+# Expand arrays to multiple lines that exceed the maximum column width.
+array_auto_expand = true
+# Collapse arrays that don't exceed the maximum column width and don't contain comments.
+array_auto_collapse = false
+# Omit white space padding from single-line arrays
+compact_arrays = true
+# Omit white space padding from the start and end of inline tables.
+compact_inline_tables = false
+# Maximum column width in characters, affects array expansion and collapse, this doesn't take whitespace into account.
+# Note that this is not set in stone, and works on a best-effort basis.
+column_width = 160
+# Indent based on tables and arrays of tables and their subtables, subtables out of order are not indented.
+indent_tables = false
+# The substring that is used for indentation, should be tabs or spaces (but technically can be anything).
+indent_string = '  '
+# Alphabetically reorder keys that are not separated by empty lines.
+reorder_keys = false
```

Comment:
Don't we want that? This will be really helpful I guess.

Reply:
not really. it messes with the whole cargo file, including the `[package]` fields etc. I had a look at other substrate projects using `taplo` and they have it disabled as well. btw this tool is planning on supporting sorting `features` arrays like `std` soon 🥳